src/schema.rs

```diff
@@ -0,0 +1,27 @@
+table! {
```

Comment:
we already have a schema in `/src/sync/schema.rs`

why did you create this file?

Reply:
My bad, i will delete that